esmvaltool/recipes/bock20jgr/recipe_bock20jgr_fig_1-4.yml

```
@@ -447,8 +445,7 @@ diagnostics:
         end_year: 1999
         additional_datasets: *cmip3_tas
     additional_datasets:
-      - {dataset: GPCP-SG, project: obs4MIPs, level: L3, version: v2.3,
-         tier: 1, mip: Amon}
+      - {dataset: GPCP-V2.3, project: obs4MIPs, level: L3, tier: 1, mip: Amon}
```

To make it possible to find the data, 
```suggestion
      - {dataset: GPCP-V2.3, project: obs4MIPs, level: L3, tier: 1, mip: Amon, frequency='mon'}
```
is needed to avoid that frequency is overwritten with `'mo'` from line 442 above.

---

It would be good to change that for the other observational datasets used in combination with CMIP3 data too.